src/main/MainScreenContainer.js

```diff
@@ -9,6 +9,13 @@ import { getAuth } from '../account/accountSelectors';
 import MainScreen from './MainScreen';
 
 class MainScreenContainer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      isFromReply: false,
```

Comment:
This way of implementing this feature is not ideal.
I also don't think the naming 'isFromReply' is clear enough.